### PR TITLE
Fels create category vicente

### PR DESCRIPTION
**Pull request by AubreyV**
If a logged in user is an admin, the user should see a **Create a Category** button beside the **All Categories** text. If not, just the list of all categories.

To set a user as an admin, change the **is_admin** property of the user to 1.

### CODE REVIEW

File: app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -47,7 +47,17 @@ public function create()
...
-        //
+        $this->validate(request(), [
+            'title' => 'required',
+            'desc'  => 'required',
+        ]);
+
+        $category = Category::create([
+            'title' => $request->title,
+            'desc'  => $request->desc,               
+        ]);
```
Comment (jeremiahC) [inline app/Http/Controllers/CategoryController.php]: can you put this in model.

File: app/Category.php [hunk 2]
```diff
@@ -6,5 +6,12 @@
...
 class Category extends Model
 {
-    
+    protected $fillable = [
+        'title', 'desc'
+    ];
+
+    public function createCategory($request)
+    {
+    	return $this->fill($request->all())->save();
```
Comment (jeremiahC) [inline app/Category.php]: fix indentation

Comment (AubreyV) [reply app/Category.php]: fixed

**Comment by jeremiahC** [comment]
if i login as admin i cannot see admin privileges like create button for category
[Screenshot from Gyazo](https://gyazo.com/5ef96d1525fdaa154386a248b10527e9)

**Review by jeremiahC — APPROVED** [review]
I approve this PR

**Pull request merged**